pydantic/fields.py

```diff
@@ -752,6 +755,7 @@ def Field(  # noqa: C901
         decimal_places: Maximum number of decimal places allowed for numbers.
         union_mode: The strategy to apply when validating a union. Can be `smart` (the default), or `left_to_right`.
             See [Union Mode](../concepts/unions.md#union-modes) for details.
+        fail_fast: If `True`, validation will stop on the first error. If `False`, all validation errors will be collected.
```

Comment:
I think we should clarify that this is for iterable types throughout the docs in this PR

Reply:
Makes sense, fixed